src/_product-updates/2025-09-29-documentation-enhancements-and-fixes.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Documentation enhancements and fixes
+tags: [Improvement]
+---
+While achieving milestones on larger projects, we’ve also made many quality-of-life enhancements and bug fixes. Here's the changelog:
+
+Enhancements:
+- [Custom meta tags](/help/specification-support/meta-tags/) can now be added to the head of your documentation,
+- Dead anchors now redirect to their parent operation instead of the top of the documentation,
+- In Embed mode, the search placeholders now specify their context (« Search API documentation » vs « Search hub »), so users can differentiate the global search from the documentaion portal search, 
+- A confirmation modal has been added to the API Explorer before sending destructive requests to avoid unwanted losses,
+- The scrolling experience has been globally reworked to ensure its stability,
+- When using groups to categorize your API documentation inside your hub, you now have the option to only display the ones from the same category in the documentation selector of a documentation,
```

Comment:
```suggestion
- Hub categories can now filter the documentation selector in the top navigation bar. This way, readers only see relevant docs from the same category, improving navigation for them and giving admins clearer control over doc organization,
```

Reply:
In any case, this also deserves a doc update, and linking to it there should be helpful https://docs.bump.sh/help/hubs/create-and-manage-hubs/#group-by-categories (then we can have longer texts, screenshots,etc.)